modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -813,9 +812,9 @@ public void testOneNodeRestartWithGap() {
 
         assertNotNull(table);
 
-        assertThrowsWithCause(() -> table.keyValueView().get(null, Tuple.create().set("id", 0)), TimeoutException.class);
+        assertThrows(TransactionException.class, () -> table.keyValueView().get(null, Tuple.create().set("id", 0)));
```

Comment:
Should we check here the cause of an exception?

Reply:
I don't think so, because several different causes are possible: ReplicationTimeout, TimeoutException, ReplicaUnavailable, ConnectRefuse, etc.